dashboard/config/levels/custom/sketchlab/allthethings sketchlab 2.level

```diff
@@ -0,0 +1,384 @@
+            "text": "hey, it's a level with starter ! with an image!",
+            "textAlign": "left",
+            "verticalAlign": "top",
+            "containerId": null,
+            "originalText": "hey, it's a level with starter ! with an image!",
+            "lineHeight": 1.25,
+            "baseline": 18
+          },
+          {
+            "type": "image",
+            "version": 75,
+            "versionNonce": 1024528498,
+            "isDeleted": false,
+            "id": "jeDLwFe6n1WVBKuSfaJ5_",
+            "fillStyle": "solid",
+            "strokeWidth": 2,
+            "strokeStyle": "solid",
+            "roughness": 1,
+            "opacity": 100,
+            "angle": 0,
+            "x": 394.18359375,
+            "y": 324.87890625,
+            "strokeColor": "transparent",
+            "backgroundColor": "transparent",
+            "width": 262.625,
+            "height": 396,
+            "seed": 941131164,
+            "groupIds": [
+
+            ],
+            "frameId": null,
+            "roundness": null,
+            "boundElements": [
+
+            ],
+            "updated": 1759431065884,
+            "link": null,
+            "locked": false,
+            "status": "pending",
+            "fileId": "Y7cl8ScCeGFydcxrvUgrN81GFg7zd0-r1YYCd57L",
+            "scale": [
+              1,
+              1
+            ]
+          },
+          {
+            "id": "lg0vwJMnRn36c9r5srB_k",
+            "type": "arrow",
+            "x": 527.1417748673002,
+            "y": 203.5316709606096,
+            "width": 0,
+            "height": 85.642144723688,
+            "angle": 0,
+            "strokeColor": "#f08c00",
+            "backgroundColor": "transparent",
+            "fillStyle": "solid",
+            "strokeWidth": 2,
+            "strokeStyle": "solid",
+            "roughness": 1,
+            "opacity": 100,
+            "groupIds": [
+
+            ],
+            "frameId": null,
+            "roundness": {
+              "type": 2
+            },
+            "seed": 1899639346,
+            "version": 34,
+            "versionNonce": 530942446,
+            "isDeleted": false,
+            "boundElements": null,
+            "updated": 1759431071849,
+            "link": null,
+            "locked": false,
+            "points": [
+              [
+                0,
+                0
+              ],
+              [
+                0,
+                85.642144723688
+              ]
+            ],
+            "lastCommittedPoint": null,
+            "startBinding": null,
+            "endBinding": null,
+            "startArrowhead": null,
+            "endArrowhead": "arrow"
+          }
+        ],
+        "appState": {
+          "gridSize": null,
+          "viewBackgroundColor": "#ffffff",
+          "scrollX": 126.90682298489048,
+          "scrollY": 9.062701205982535,
+          "zoom": {
+            "value": 0.9397297208011152
+          }
+        },
+        "files": {
```

Comment:
This...doesn't feel great to have in a non-git LFS part of our repo. Loading images from external locations doesn't appear to be supported out of the box by Excalidraw ([a bit more discussion in this GitHub Issue](https://github.com/excalidraw/excalidraw/issues/9491)), but I played a little bit today with loading files from external sources and encoding them before passing them to Excalidraw.

We could in theory do this instead of storing images as base64 strings. It would complicate things a bit to do that, but it might be worthwhile.

Reply:
Yeah I don't love this but it seems ok for now.